controllers/ilightthat.xcontroller

```diff
@@ -0,0 +1,27 @@
+<Vendor Name="ILightThat">
+    <AbstractVariant Name="BaseBaldrickSettings">
+		<SupportsPixelPortBrightness/>
+		<SupportsPixelPortColourOrder/>
+        <SupportsUpload/>
+        <MaxInputUniverses>9</MaxInputUniverses>
+        <MaxSerialPortChannels>0</MaxSerialPortChannels>
+        <MaxPixelPortChannels>6000</MaxPixelPortChannels>
```

Comment:
change MaxPixelPortChannels to 2250 as this would be 750 pixels per port. Then you can remove the check from the CPP code.

Reply:
Stupidly misinterpreted that one - Thanks :)

Revised to 2250, removed CPP check.